### PR TITLE
InternLM Summer Camp3

### PR DESCRIPTION
**Pull request by boshallen**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by zhyncs — COMMENTED** [review]
@boshallen Do we really need to add this to the README, or is it enough to just raise an issue and **pin** it?

**Comment by zhyncs** [comment]
ref https://github.com/InternLM/lmdeploy/issues/2021

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ ______________________________________________________________________
 <details open>
 <summary><b>2024</b></summary>
 
+- \[2024/07\] 🔥 Join the InternLM [Summer Camp3](https://www.wjx.cn/vm/PvefmG2.aspx?sojumpparm=github) now!
```
Comment (zhyncs) [inline README.md:29]: May you resolve the conflict?

**Review by zhyncs — CHANGES_REQUESTED** [review]

**Comment by lvhan028** [comment]
Vote for the pinned issue

**Pull request closed without merging**